target/linux/ramips/dts/mt7628an_creality_wb-01.dts

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "creality,wb-01", "mediatek,mt7628an-soc";
+	model = "Creality WB-01";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-upgrade = &led_status;
+		led-running = &led_status;
+		label-mac-device = &wmac;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "yellow:wlan2g";
```

Comment:
Please use `function` (`LED_FUNCTION_WLAN_2GHZ`) and `color` (`LED_COLOR_ID_YELLOW`) instead of `label`.

Applies to other LEDS as well.

Reply:
hi @rmilecki 
i was hoping that this device could be backported to 23.05, but with this new LED semantic ..
anyway ... done as you suggested :)
hope it is OK now